compiler/optimizer/VPHandlers.cpp

```diff
@@ -5239,6 +5238,98 @@ TR::Node *constrainCall(TR::ValuePropagation *vp, TR::Node *node)
       bool isFloatAbs =   (symbol->getRecognizedMethod() == TR::java_lang_Math_abs_F);
       bool isDoubleAbs =  (symbol->getRecognizedMethod() == TR::java_lang_Math_abs_D);
 
+      bool isGetModifiersImpl = (symbol->getRecognizedMethod() == TR::java_lang_Class_getModifiersImpl);
```

Comment:
This would probably look nicer as a switch case and would run a little bit faster too.